script.js

```diff
@@ -63,6 +68,8 @@ const functions = [
  * targets: The query selector to apply the function to
  */
 const startFunctions = [
+  disableAutoplay,
```

Comment:
Replace with the object in the functions variable

Reply:
Fixed it!